### DISCUSSION TITLE
32bit - start full screen

### DISCUSSION THREAD

**Discussion by AndyIreland** [cedricp/ddt4all#943] [Q&A]
hello can someone fix 32bit version 2.1.1 to open full screen as it opens like this and the mouse pointer does not correspond to the scroll bars etc 
Thank you .

![image](https://github.com/cedricp/ddt4all/assets/49172217/15d9f401-c956-4d03-a1e4-1de4eddc5aa2)

you have to move the mouse across to the right to scroll the scroll bar up and down

@Furtif  @cedricp   ?

**Comment by Furtif** [comment]
🙄

**Comment by AndyIreland** [comment]
So I downloaded this Main.py and replace it into the DDT4ALL Folder (program files 386) - but then I cannot get DDT4ALL started at all then
![image](https://github.com/cedricp/ddt4all/assets/49172217/f6d78308-cd69-4aaf-a226-d780ef636d18)

**Reply by Furtif** [reply]
I don't know what you're doing, the repo has a lot more files, it's a submodule, a readme how to make a clone correctly, it's a description... they get problems where there aren't any.

**Comment by AndyIreland** [comment]
ok I will try the 64bit version and see if that opens up the main screen window in fullscreen

**Reply by Furtif** [reply]
🤷🏻‍♂️💁🏻‍♂️

**Comment by Furtif** [comment]
fine ?

**Reply by AndyIreland** [reply]
no, still the same I have to either reduce the window size or go back to using this version DDT4ALL 'All Cars' from here which works flawlessly https://www.drive2.ru/b/498093336985338243/
I want to stay on 32bit version if I can rather than 64bit version
